catalogd/cmd/catalogd/main.go

```diff
@@ -145,12 +146,16 @@ func main() {
 	}
 
 	if (certFile != "" && keyFile == "") || (certFile == "" && keyFile != "") {
-		setupLog.Error(nil, "unable to configure TLS certificates: tls-cert and tls-key flags must be used together")
+		setupLog.Error(errors.New("missing TLS configuration"),
+			"message", "tls-cert and tls-key flags must be used together",
```

Comment:
```suggestion
			"tls-cert and tls-key flags must be used together",
```
For `log.Error()`, the first argument is the `error`, the second is the `message`, and subsequent arguments are label+value pairs.

Reply:
Oh thank you for getting it.
It was a silly mistake